MobileFilter/sections/specific_web.txt

```diff
@@ -4257,6 +4257,13 @@ popularmechanics.com#?#section > div[class*=" "]:has(> div[class]:first-child +
 veranda.com,wetter.com#?#div[class^="sc-"]:has(> div.ad-container)
 wetter.com#?#.som-wrapper:has(> div[data-slotname])
 wetter.com#?#.clickperformance-wrapper:has(> .ad-flex)
+wetter.com#?#.mbanner-fixed:remove()
```

Comment:
why extended CSS? `wetter.com##.mbanner-fixed` works fine on my Android device.

Reply:
Is non-extended CSS faster? If not, then, I think, `:remove()` is better because we can remove the elment altogether and forget about it. Plus, while doing the task, I saw some elements on this site which have `display: flex !important` that override our `display: none !important` :)